alonzo/impl/src/Cardano/Ledger/Alonzo/PParams.hs

```diff
@@ -24,30 +27,30 @@ module Cardano.Ledger.Alonzo.PParams
     PParamsUpdate,
     emptyPParamsUpdate,
     updatePParams,
+    EraIndependentPP,
+    Hashable (..),
+    hashLanguagePP,
   )
 where
 
-import Cardano.Binary
-  ( FromCBOR (..),
-    ToCBOR (..),
-    decodeWord,
-    encodeListLen,
-    encodeMapLen,
-    encodeWord,
-  )
+import Cardano.Binary (FromCBOR (..), ToCBOR (..), decodeWord, encodeListLen, encodeMapLen, encodeWord, serialize')
```

Comment:
maybe it's just me, but when the imports get past three or so, I prefer them vertical. it makes it easier to see what is changing in diffs.

Reply:
done